regexes/oss.yml

```diff
@@ -1233,15 +1248,15 @@
   name: 'iOS'
   version: ''
 
-- regex: 'iOS/Version ([\d\.]+)'
+- regex: 'iOS[ /]Version ([\d\.]+)'
   name: 'iOS'
   version: '$1'
 
 - regex: 'Sonos/.+\(ICRU_'
   name: 'iOS'
   version: ''
 
-- regex: 'CaptiveNetworkSupport|AirPlay'
+- regex: 'CaptiveNetworkSupport|AirPlay|.*[ \.\-/]iOS'
```

Comment:
`.*[ ./-]iOS`

EA:
Redundant character escape '\.' in RegExp 
 Inspection info: Reports redundant character escape sequences that can be replaced with unescaped characters preserving the meaning. Many escape sequences that are necessary outside of a character class are redundant inside square brackets [] of a character class.
Although unescaped opening curly braces { outside of character classes are allowed in some dialects (JavaScript, Python, and so on), it can cause confusion and make the pattern less portable, because there are dialects that require escaping curly braces as characters. For this reason the inspection does not report escaped opening curly braces.
Example:
  \-\;[\.]
After the quick-fix is applied:
  -;[.]
  
  
  This is not a mistake, we will put everything in order in the future